CHANGELOG.md

```diff
@@ -2,6 +2,43 @@
 
 All notable changes to the "vscode-azurearmtools" extension will be documented in this file.
 
+## Version 0.10.0
+
+### Added
+
+- Improved parameter file support:
+  - Completions for missing and new parameter values (type double quote or CTRL+SPACE)
+  - Quick fix and code actions for adding missing parameter values
+  - Rename parameters across template and parameter file
+  - Go To Definition and Find References for parameters across both files
+
+- Insert item... (variables, parameters, resources, functions, outputs) **implemented by Nils Hedström @nilshedstrom, thanks!**
+Look for this in the editor context menu or the ARM Template Outline view!
+- Completions for resourceId arguments based on resources found inside the template
+- New `Application Gateway` and `Application Gateway and Firewall` snippets (thanks Emmanuel Auffray @ManuInNZ!)
+
+### Changed
+
+- Suggestion: Auto-complete should not insert "()" [#501](https://github.com/microsoft/vscode-azurearmtools/issues/501)
+- Removed `arm-param-value` snippet in parameter files (replaced by new parameter auto-completions)
+- Add lightbulbs to increase discoverability of parameter/variable renames
```

Comment:
Maybe include a "thanks"?

Reply:
Ugh, missed that, sorry!  Will add...

Reply:
Ugh, missed that, sorry!  Will add...

Reply:
https://github.com/microsoft/vscode-azurearmtools/pull/703  

I didn't miss anything else, did I?

Reply:
As far as I know you did not miss anything else :)

Reply:
Good.  Thanks for letting us know.